articles/2013-07-12-binaergewitter-talk-number-58-schoepfungshoehe-nicht-erreicht.markdown

```diff
@@ -13,16 +13,16 @@ audioformats:
 ---
 Mit Pfleidi, Felix und Ingo und auch gar nicht so lang aber wieder dumm, unlustig und geschmacklos.
 
-##Intro ```00:00:00```  
```

Comment:
:tada: niice

Reply:
dann bitte nen chapter file draus machen ... :) einfach löschen wäre schade

Reply:
wo kann ich die chapter file hinlegen damit es dazu wird?

Reply:
Hier sollte das glaub ich sein dann als txt file http://download.binaergewitter.de/

Reply:
ok mach ich. muss das auch hier irgendwo mit rein in das repo? oder haben wir chaptermarks nicht unter versionskontrolle?

Reply:
leider nein haben wir nicht die leben nur auf dem download server

Reply:
waer auch mal ein projekt :D

Reply:
> We are living in the DANGER ZONEEEEEE!

Reply:
```
00:00:17.444 Hallihallo und Herzlich Willkommen
00:05:27.444 Blast from the Past
00:11:41.444 Toder der Woche
00:17:00.444 Untoter der Woche
00:17:05.444 Open Leaks
00:17:57.444 Tizen
00:20:31.444 News
00:20:40.444 Valve hat einen Github Account für SDKs
00:27:20.444 Kein Urheberrechtsschutz für pr0n
00:31:30.444 Sähroboter
00:34:29.444 Sky will Werbung in die Brains projizieren
00:39:31.444 Untertitelseite offline wegen Razia
00:42:45.444 Oracle ändert Lizenz von BerkleyDB zu AGPL
00:45:27.444 Endlich saubere Energie
00:48:00.444 Firefox Phone in Deutschland
00:49:58.444 Chrome 28 kommt mit Blink
00:51:35.444 3D-Kopierer/Drucker vom Discounter
00:56:21.444 FFmpeg mit OpenCL
01:02:12.444 Lesefoo
01:02:36.444 APIGeee Web API Design Ebook
01:14:18.444 Benchmarking Ruby Code
01:15:25.444 A Statistical Analysis of Nerf Blasters and Dars
01:18:12.444 Mimimi der Woche
01:30:25.444 Picks
01:40:00.444 Ende
```

die file liegt jetzt auch unter download.binaergewitter.de/2013-07-04.Binaergewitter.Talk.58.chapters.txt

Reply:
:+1: habs im post verlinkt damit sollte es dann auftauchen